src/webview/MessageList.js

```diff
@@ -261,23 +245,15 @@ class MessageList extends Component<Props> {
         // Otherwise (for iOS), return a closure that evaluates to `true` _exactly
         // once_, and even then only if the URL looks like what we're expecting.
         let loaded_once = false;
-        // The baseUrl, with its relative portion (if any) stripped.
-        const baseUrlTail = baseUrl.replace(/^\.\//, '');
-        // Disallow such monstrosities as `evilsite.com/?./webview/index.html`.
-        const unsafeUrlRegex = /[&?]/;
         return (url: string) => {
-          if (!loaded_once) {
-            // The exact URL that will be loaded could be determined statically on
-            // Android. On iOS, though, it involves some unpredictable UUIDs which
-            // RN provides no good interface to. (`react-native-fs` is awful.)
-            if (
-              url.startsWith('file://')
-              && url.endsWith(baseUrlTail)
-              && !unsafeUrlRegex.test(url)
-            ) {
-              loaded_once = true;
-              return true;
-            }
+          const parsedUrl: WhatwgURL | void = tryParseUrl(url);
```

Comment:
This annotation isn't needed, is it? It's just the plain return type of `tryParseUrl`.